docs/root/version_history/current.rst

```diff
@@ -12,6 +12,7 @@ Minor Behavior Changes
 ----------------------
 *Changes that may cause incompatibilities for some users, but should not for most*
 
+* access_log: add GRPC_STATUS_NUMBER format command operator.
```

Comment:
I think this belongs farther down in the "New Features" section. You can see there are already some other new log operators there.

Reply:
whoops – fixed!